src/js/language.js

```diff
@@ -77,6 +77,13 @@ export const translateConfirmWord = new Map([
       en: 'Are you sure you want to delete all whitelist?',
       ja: '本当にすべて削除しますか？'
     }
+  ],
+  [
+    'whiteListDomainClearConfirm',
+    {
+      en: 'THe domain exists on whitelist.\n Are you sure close them?',
```

Comment:
TheよりはThisのほうが状況に合いそう。
開業直後の空白はなくても良さそう。

```suggestion
      en: 'This domain exists on whitelist.\nAre you sure close them?',
```